src/frequenz/sdk/timeseries/formula_engine/_formula_engine.py

```diff
@@ -313,7 +313,7 @@ async def _run(self) -> None:
             try:
                 msg = await evaluator.apply()
             except asyncio.CancelledError:
-                _logger.exception("FormulaEngine task cancelled: %s", self._name)
+                _logger.info("FormulaEngine task cancelled: %s", self._name)
```

Comment:
Not sure if this is really useful information to have unless one is debugging. I would make it a `debug`.